src/Servers/Connections.Abstractions/src/FileHandleEndPoint.cs

```diff
@@ -8,10 +8,11 @@ namespace Microsoft.AspNetCore.Connections
 {
     public class FileHandleEndPoint : EndPoint
     {
-        public FileHandleEndPoint(ulong fileHandle, FileHandleType fileHandleType)
+        public FileHandleEndPoint(ulong fileHandle, FileHandleType fileHandleType, bool ownsHandle = true)
```

Comment:
The binary breaking change guidelines say this should be a new overload rather than an optional parameter.

Reply:
Yea I wasn't sure about that. Will change it.